assets/scss/components/_dropdown.scss

```diff
@@ -88,9 +98,11 @@ $spaceFromTrigger: $space-3 + $triangleSize;
 	transform: translateX(-50%);
 
 	&:before {
-		transform: translateX(50%);
+		left: 50%;
```

Comment:
Should this `left` property be conditionally applied for the `--noPortal` variant only?

Reply:
Ok, so it looks like JS overrides this when the dropdown is in a Portal... Just confirming this is right

Reply:
yes, it does